server.R

```diff
@@ -11,6 +11,10 @@ shinyServer(function(input, output, session) {
   
   ### GDPR
   
+  google_analytics_header_server(id = "analytics")
+
+  ### View the full update history
+  
   showModal(
```

Comment:
This modal needs to be removed, otherwise you get a double warning, which is an issue when the new model is rejected, this one still shows, looking like you have no option but to consent.

Reply:
Not sure how I left that in. Removed now.